gateway/ln-gateway/src/bin/gatewayd.rs

```diff
@@ -10,6 +10,7 @@ use fedimint_core::core::{
 use fedimint_core::module::registry::ModuleDecoderRegistry;
 use fedimint_core::module::DynModuleGen;
 use fedimint_core::task::TaskGroup;
+use fedimint_server::logging::TracingSetup;
```

Comment:
Separate issue since the dep was already there, but `gatewayd` should not depend on `fedimint-server` imo.